src/js/hca/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { focusElement } from '../../common/utils/helpers';
+import OMBInfo from '../../common/components/OMBInfo';
+import FormTitle from '../../common/schemaform/FormTitle';
+import SaveInProgressIntro, { introActions, introSelector } from '../../common/schemaform/SaveInProgressIntro';
+
+class IntroductionPage extends React.Component {
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="Apply for health care benefits"/>
+        <p>Equal to VA Form 10-10EZ (Application for Health Benefits).</p>
+        <SaveInProgressIntro
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          resumeOnly
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}/>
+        <div className="process schemaform-process">
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>To fill out this application, you’ll need your:</h6></div>
+              <ul>
+                <li>Social Security number (required)</li>
```

Comment:
Should 'Social Security' be capitalized? I recall it being lower case elsewhere? @jbalboni @peggygannon ?

Reply:
Social Security is capitalized, but number is lower case.

Reply:
@cehsu Yes, Social Security should be capped.